stub_uploader/get_version.py

```diff
@@ -83,28 +60,90 @@ def check_exists(distribution: str) -> bool:
     raise ValueError("Error while verifying existence")
 
 
-def main(typeshed_dir: str, distribution: str, version: Optional[str]) -> int:
-    """A simple function to get version increment of a third-party stub package.
-
-    Supports basic reties and timeouts (as module constants).
-    """
-    pypi_versions = fetch_pypi_versions(distribution)
-    if not version:
-        # Use the METADATA.toml version, if not given one.
-        version = read_base_version(typeshed_dir, distribution)
-    matching = [v for v in pypi_versions if v.startswith(f"{version}.")]
-    if not matching:
-        return -1
-    increment = max(int(v.split(".")[-1]) for v in matching)
-    return increment
-
-
-if __name__ == "__main__":
-    parser = argparse.ArgumentParser()
-    parser.add_argument("typeshed_dir", help="Path to typeshed checkout directory")
-    parser.add_argument("distribution", help="Third-party distribution to build")
-    parser.add_argument(
-        "version", nargs="?", help="Base version for which to get increment"
+def extend_to_specificity(ver: list[int], specificity: int) -> None:
+    ver.extend([0] * (specificity - len(ver)))
```

Comment:
Maybe add an assertion?

```suggestion
def extend_to_specificity(ver: list[int], specificity: int) -> None:
    assert specificity >= len(ver)
    ver.extend([0] * (specificity - len(ver)))
```

Reply:
That's not guaranteed to be the case (and this behaviour is tested). I renamed the function to try to make that clearer.